backend/src/tests.py

```diff
@@ -148,3 +150,26 @@ def test_rrule_validation_failure(self):
         }
         response = self.client.post("http://testserver/api/rules", params={"userid": "testuser"}, json=body)
         self.assertEqual(response.status_code, 400)
+
+    def test_future_start_end_dates(self):
+        response = self.client.post("http://testserver/api/transactions", params={"userid": "testuser", "currentBalance": "0", "startDate": "2020-1-1", "endDate": "2021-1-1"})
+        self.assertEqual(response.status_code, 400)     
+
+    def test_start_date_preceeds_end_date(self):
+        now = datetime.now().date()
+        endDate = now - relativedelta(days=1)      
+        response = self.client.post("http://testserver/api/transactions", params={"userid": "testuser", "currentBalance": "0", "startDate": now.strftime("%Y-%m-%d"), "endDate": endDate.strftime("%Y-%m-%d")})
+        self.assertEqual(response.status_code, 400)
+
+    def test_future_limit_start_date(self):
+        now = datetime.now().date()
```

Comment:
You're probably looking for `date.today()`

Reply:
I've modified the code to use "date.today()"

Reply:
@rscronce did you forget to push? I don't see the change.

Reply:
(And in the API code itself)

Reply:
Done: 4863f150d5dfd245752e298fce3f2ed69da6b015